lib/redux/reducers/devices.js

```diff
@@ -25,6 +25,21 @@ const devices = {
     source: {type: 'device', driverId: 'Medtronic'},
     enabled: {mac: true, win: true}
   },
+  medtronic600: {
+    instructions: 'Connect your Bayer Contour Next Link 2.4 to your computer',
```

Comment:
I agree with Lennart that the only way to distinguish this meter from the other BCN Link is the "2.4" model number. What I do think will be confusing is selecting it from the "Choose Devices" screen, where it shows up as 
<img width="414" alt="screen shot 2017-01-16 at 13 31 07" src="https://cloud.githubusercontent.com/assets/418233/21984936/56decec4-dbf0-11e6-803b-68b1ac33527e.png">

I'll refer this to #design on Slack.